media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild

```diff
@@ -7,7 +7,7 @@ inherit desktop xdg-utils
 
 DESCRIPTION="A client software for quality voice communication via the internet"
 HOMEPAGE="https://www.teamspeak.com/"
-SRC_URI="https://files.teamspeak-services.com/teamspeak/teamspeak-client.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://files.teamspeak-services.com/pre_releases/client/${PV}/teamspeak-client.tar.gz -> ${P}.tar.gz"
```

Comment:
This won't work, as ```PV=5.0.0-beta70```, but link was prior ```5.0.0_beta70```. So you need to adjust this, something like ```${PV/-/_}```, this should work.

Reply:
The issue was with the previous name. This works:
https://files.teamspeak-services.com/pre_releases/client/5.0.0-beta70/teamspeak-client.tar.gz
This doesn't:

Reply:
Ah! Thank you, so I did missunderstood. Then its fine of course.

Reply:
Actually, no, this is not allowed. As per PMS, you need to use ```_beta``` with underscore as suffix. I haven't seen, that you changed the ebuild name here. The suffix ```-beta``` is not allowed.

Reply:
Should be okay now?